### ISSUE TITLE
freezes in-car browser 

### ISSUE THREAD

**Issue by ulrichard** [shesek/spark-wallet#114]
Spark used to run just fine in the Tesla in-car browser for a couple of months.
But since I upgraded the spark server, trying to load the website in the car always freezes the browser. The loading indicator turns for a couple of seconds and then stalls.

**Comment by shesek** [comment]
Are you able to access it properly from a different (more typical, ie your laptop or phone) device?

**Comment by ulrichard** [comment]
yes, on all other devices it still works. It is only the car browser that freezes lately.
Problem is, that since the car has free internet, I usually don't buy mobile data for the phone any more. It used to work perfectly for buying BTC in person for cash.

**Comment by shesek** [comment]
It's hard to diagnose the bug without being able to reproduce it. Do you know which browser is running in there? Is it possible to somehow run it without owning a Tesla?

**Comment by ulrichard** [comment]
I didn't find any emulator, only people asking for one.
What I know is that recent versions are based on Chromium, older ones used some Qt web control.
